spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -37,15 +37,25 @@
   context 'autocomplete results' do
     it 'returns collection of matches' do
       create(:version, :production)
-      7.times { create(:institution, :contains_harv, approved: true) }
+      2.times { create(:institution, :start_like_harv, approved: true) }
       get :autocomplete, term: 'harv', version: 'production'
+      expect(JSON.parse(response.body)['data'].count).to eq(2)
+      expect(response.content_type).to eq('application/json')
+      expect(response).to match_response_schema('autocomplete')
+    end
+
+    it 'limits results to 6' do
+      create(:version, :production)
+      7.times { create(:institution, :start_like_harv, approved: true) }
+      get :autocomplete, term: 'harv', version: 'production'
+      expect(JSON.parse(response.body)['data'].count).to eq(6)
```

Comment:
Does this have to be `6` based on test fixtures or something? 

I am getting a whiff of mystery guest here, not sure what "harv" means (even though I could guess). 

Functionally it looks good, just a readability nitpick

Reply:
That's what the [model method](https://github.com/department-of-veterans-affairs/gibct-data-service/blob/9e2dbed9455fe491e803a68243408899458a59e0/app/models/institution.rb#L216) defaults to.